package.json

```diff
@@ -13,7 +13,7 @@
     "docker:clean": "docker-compose down -v",
     "web:dev": "lerna run dev --parallel --scope @metafam/web --include-dependencies",
     "web:build": "lerna run build --scope @metafam/web --include-dependencies --stream",
-    "backend:dev": "lerna run dev --parallel --scope @metafam/backend --include-dependencies",
+    "backend:dev": "lerna run dev --parallel --scope @metafam/backend",
```

Comment:
Yeah, I made this same change at some point but then ended up reverting it... can't remember why.